### DISCUSSION TITLE
Hide specific root object during serialization and show only it's children properties in yaml

### DISCUSSION THREAD

**Discussion by aksh1998** [aaubry/YamlDotNet#692] [Q&A]
Hi,
I am using one object model

public class MainModel
    {
        public FirstChildModel firstChild { get; set; }
        public SecondChildModel secondChild { get; set; }
    }
public class FirstChildModel 
    {
        public string firstProp { get; set; }
        public string secondProp { get; set; }
    }
public class SecondChildModel 
    {
        public string name { get; set; }
        public bool isValid { get; set; }
    }

and I am getting it's respective yaml as
![image](https://user-images.githubusercontent.com/52662888/171336645-e01655c1-fe9b-434f-a7ca-cbd3b141ed90.png)

But is there any way so that i can hide specific root property and show it's respective children properties like

![image](https://user-images.githubusercontent.com/52662888/171336728-3684c0f4-372b-44cd-a7ce-33861659ea8a.png)

**Comment by EdwardCooke** [comment]
If I understand your question correctly, no, there is not a way of doing that natively within yamldotnet.
